bin/ocamlformat.ml

```diff
@@ -75,7 +75,7 @@ let action, opts = Conf.action ()
 
 ;;
 match action with
-| Inplace inputs ->
+| `Ok (Inplace inputs) ->
```

Comment:
What do you think of moving the action cases to a `action -> int` function, returning the exit status ?

Reply:
I did something slightly different, returning a result type instead, it's better I think.